.projenrc.js

```diff
@@ -1,9 +1,12 @@
-const { awscdk } = require('projen');
+const { awscdk, DependencyType } = require('projen');
+
+const CDK_VERSION = '2.20.0';
```

Comment:
S3 EventBridge feature is only enabled from 2.20+

Reply:
The version of CDK is actually [already upgraded](https://github.com/awslabs/aws-ddk/commit/c6343fcd661e7dd157c1f3acbcc858472ccafecb), so you can just merge from `typescript-conversion`.